tasklib/backends.py

```diff
@@ -320,6 +321,10 @@ def merge_with(self, path, push=False):
     def undo(self):
         self.execute_command(['undo'])
 
+    @lru_cache(maxsize=128)
```

Comment:
The cache doesn't seem to be cleared on `refresh`, is that intended?

Reply:
Not specifically, I guess it would make sense to clear it then!